docs/products/kafka/concepts/partition-segments.rst

```diff
@@ -0,0 +1,65 @@
+Partition segments
+==================
+
+Apache Kafka divides topics partition data into **segment** files (with ``.log`` suffix) stored on the file system. Each segment file is named using the offset of the first message (a.k.a. **base offset**) contained. E.g. the segment file ``04.log`` contains the message with offset ``4`` as first entry. 
+
+The last segment in the partition is called the **active segment** and it is the only segment to which new messages are appended to.
+
+In the example above you a topic partition is divided into multiple segments, with ``06.log`` being the active one:
```

Comment:
```suggestion
In the example above the topic partition is divided into multiple segments, with ``06.log`` being the active one:
```
do we need to specify which topic partition is? I think it was meant to be **your** in any case.